webserver/codechecker/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -------------------------------------------------------------------------
```

Comment:
You added some extra `PYTHONPATH` values previously to run the test was that because codechecker directory was not a package? This is changed here.

Reply:
We added extra values to `PYTHONPATH`, because multiple parts of the CodeChecker (`analyzer`, `server` and `client`) have the same library called `codechecker` and we would like to use the merged one in the package.